packages/node_modules/@ciscospark/react-component-activity-text/src/index.js

```diff
@@ -38,19 +43,22 @@ export default function ActivityText(props) {
       h5: [],
       h6: []
     }, [], content);
-    const htmlContent = {__html: cleaned};
+    const linkifiedContent = linkifyHtml(cleaned, linkifyOptions);
+    const htmlContent = {__html: linkifiedContent};
     return (
       <div
         className={classNames(`ciscospark-activity-text`, styles.activityText)}
         dangerouslySetInnerHTML={htmlContent}
       />
     );
```

Comment:
Probably don't need this return since the next block returns the same jsx. Just need to set the HTML content based on if the HTML content is available or not